kie-server-parent/kie-server-remote/kie-server-jms/src/main/java/org/kie/server/jms/KieServerMDB.java

```diff
@@ -313,11 +314,15 @@ public void onMessage(Message message) {
             }
 
         } finally {
-            try {
-                closeConnectionAndSession(connect);
-            } catch (JMSRuntimeException runtimeException) {
-                logger.error("Error while attempting to close connection/session",runtimeException);
-            } finally {
+            if (connect != null) { // Only attempt to close the connection/session if they were actually created
```

Comment:
I'd probaby have done a debug log if conncetion == null. that included the "interactionPattern" and  "UPPER_LIMIT_REPLY_INTERACTION_PATTERNS" values. So in debug we know the onMessage was FireAndForge